docs/dataformats/index.rst

```diff
@@ -25,32 +25,32 @@ Overview
------------------ ---------------- -------------------------------------- ----------------------------
-Exposure          EXPOSURE_3D      `~gammapy.data.SpectralCube`?          `GCTACubeExposure`_
------------------ ---------------- -------------------------------------- ----------------------------
-Background        BACKGROUND_3D    `~gammapy.data.SpectralCube`?          `GCTABackground3D`_
-Background        BACKGROUND_1D    N/A                                    `GCTAModelRadialAcceptance`_
-Exposure          ???              `~gammapy.data.SpectralCube`?          `GCTACubeBackground`_
------------------ ---------------- -------------------------------------- ----------------------------
-PSF               PSF_2D           N/A                                    `GCTAPsf2D`_
-PSF               PSF_1D           `~gammapy.irf.TablePSF`                `GCTAPsfVector`_
-PSF               gtpsf_ output    `~gammapy.irf.EnergyDependentTablePSF` N/A
-PSF               ???              N/A                                    `GCTACubePsf`_
------------------ ---------------- -------------------------------------- ----------------------------
-Effective area    AEFF_2D          `~gammapy.irf.EffectiveAreaTable2D`    `GCTAAeff2D`_
-Effective area    ARF              `~gammapy.irf.EffectiveAreaTable`      `GCTAAeffArf`_
------------------ ---------------- -------------------------------------- ----------------------------
-Energy dispersion EDISP_2D         N/A                                    `GCTAEdisp2D`_
-Energy dispersion RMF              `~gammapy.irf.EnergyDispersion`        `GCTAEdispRMF`_
-================= ================ ====================================== ============================
+================= ================ ================================================ ============================
+Type              Format Name      Gammapy                                          Gammalib
+================= ================ ================================================ ============================
+Events            EVENTS           `~gammapy.data.EventList`                        GEventList_
+----------------- ---------------- ------------------------------------------------ ----------------------------
+Counts            3D               `~gammapy.data.SpectralCube`                     `GCTAEventCube`_
+Counts            Image            no class?                                        `GSkyMap`_
+Counts            PHA              `~gammapy.data.CountsSpectrumDataset`            `GPha`_
+----------------- ---------------- ------------------------------------------------ ----------------------------
+Exposure          EXPOSURE_3D      `~gammapy.data.SpectralCube`?                    `GCTACubeExposure`_
+----------------- ---------------- ------------------------------------------------ ----------------------------
+Background        BACKGROUND_3D    `~gammapy.background.models.CubeBackgroundModel` `GCTABackground3D`_
```

Comment:
I'd like to get rid of the `gammapy.background.models` sub-namespace and just have everything in `gammapy.background`. Too complicated to remember.

So please
* Remove this: https://github.com/gammapy/gammapy/blame/master/docs/background/index.rst#L36
* Add `from .models import *` here: https://github.com/gammapy/gammapy/blob/master/gammapy/background/__init__.py#L11
* Change `~gammapy.background.models.CubeBackgroundModel` to `~gammapy.background.CubeBackgroundModel` in this line.